setup.py

```diff
@@ -51,6 +51,7 @@
     'contextvars;python_version < "3.7"',
     'dataclasses;python_version < "3.7"',
     "chardet",
+    "markdown",
```

Comment:
This could be done on the frontend right?

Reply:
I think the browser needs to install a markdown extension to render it.  This PR sends rendered markdown as HTML down to the clients.

Reply:
You should be able to include a JS library for rendering markdown on frontend only. I'm fine either way.